_inc/connect-button.js

```diff
@@ -42,11 +42,17 @@ jQuery( document ).ready( function( $ ) {
 			}
 		},
 		startConnectionFlow: function() {
-			var abTestName = 'jetpack_connect_in_place_v2';
+			var abTestName = 'jetpack_connect_in_place_v3';
```

Comment:
Let's be careful about shipping that though. We're already running an A/B test which is still in progress.

Reply:
What concerns do you have here?

Reply:
I just want to make sure we are running the new test after the previous one has been concluded.

Reply:
How does creating a new test one now prevent us from concluding old test?As far as I know this will not ship anytime soon.

Reply:
> As far as I know this will not ship anytime soon.

Okay, this is good enough to confirm then.